packages/block-library/src/cover/block.json

```diff
@@ -31,6 +31,9 @@
 		},
 		"minHeight": {
 			"type": "number"
+		},
```

Comment:
It seems the name attribute is added just as a way to show that there is some media when a video background is used.
If that's the case I would prefer if we remove the name for now, and as an improvement, we add a small video element to show there is a video selected. Or we show the URL currently selected. It seems adding a new attribute just for this is not worth it.
The current version would also need some additional styles:
![image](https://user-images.githubusercontent.com/11271197/63703733-7392fe80-c821-11e9-99fe-57521e1fbb28.png)

So I would prefer if for now we just removed the name so we can advance with the PR and later we can use better UI for videos.